src/main/java/org/tdl/vireo/controller/SubmissionController.java

```diff
@@ -743,7 +743,7 @@ private void processBatchExport(HttpServletResponse response, User user, String
                         Path path = assetService.getAssetsAbsolutePath(ldfv.getValue());
                         byte[] fileBytes = Files.readAllBytes(path);
                         zos.putNextEntry(new ZipEntry(submissionName + ldfv.getFileName()));
-                        contentsText.append(ldfv.getFileName() + " bundle:LICENSE\n");
+                        contentsText.append(ldfv.getFileName()+"\n");
```

Comment:
I believe `bundle:LICENSE` (and `bundle:CONTENT` below) are supported and used by Dspace 6:

I think these two should still be supported as they are specific DSpace import helpers.

Is there some specific reason as to why these two have been removed?

see: https://wiki.duraspace.org/display/DSDOC6x/Importing+and+Exporting+Items+via+Simple+Archive+Format#ImportingandExportingItemsviaSimpleArchiveFormat-DSpaceSimpleArchiveFormat

Reply:
Since they are optional the task was deferred and set as issue 1313 pending another conversation with Stephanie.

Reply:
I could put in bundle:ORIGINAL and bundle:LICENSE but wasnt sure what others could be added.  I've retained primary:true for the primary doc.    I need to run now but can add the above in a separate PR on monday.  Other bundle suffixes could be added later.

Reply:
PR branch updated